ribs/emitters/operators/_gaussian.py

```diff
@@ -0,0 +1,48 @@
+"""Gaussian Operator"""
+import numpy as np
+
+from ribs.emitters.operators._operator_base import OperatorBase
+
+
+class GaussianOperator(OperatorBase):
+    """Args:
+        archive (ribs.archives.ArchiveBase): An archive to use when creating and
+            inserting solutions. For instance, this can be
+            :class:`ribs.archives.GridArchive`.
+        sigma (float or array-like): Standard deviation of the Gaussian
+            distribution. Note we assume the Gaussian is diagonal, so if this
+            argument is an array, it must be 1D.
+        lower_bounds (array-like): Upper bounds of the solution space. Passed in
+            by emitter
+        upper_bounds (array-like): Upper bounds of the solution space. Passed in
+            by emitter
+        seed (int): Value to seed the random number generator. Set to None to
+            avoid a fixed seed.
+      """
```

Comment:
These triple quotes should be lined up with the first ones. Definitely see if you can get yapf working in your editor; it would fix these kinds of things.